### PR TITLE
fix jekins

### PR DESCRIPTION
**Pull request by dding3**
Fix https://github.com/intel-analytics/arda-docker/issues/509

### CODE REVIEW

**Comment by dding3** [comment]
jekins pass. http://10.112.231.51:18888/view/ZOO-PR/job/BigDL-Orca-PR-Validation/114/
http://10.112.231.51:18888/view/ZOO-PR/job/BigDL-PR-Dllib-Validation/82/

File: python/orca/example/learn/bigdl/attention/transformer.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
 from tensorflow.python.keras.datasets import imdb
 from tensorflow.python.keras.preprocessing import sequence
 
-from bigdl.dllib.keras.models import Model
+from bigdl.dllib.keras.model import Model
```
Comment (jason-dai) [inline python/orca/example/learn/bigdl/attention/transformer.py:23]: This is not correct; it should be either `from bigdl.dllib.keras import Model` or `from bigdl.dllib.keras.models import Model`

File: python/orca/test/bigdl/orca/learn/spark/test_estimator_for_bigdl.py [hunk 2]
```diff
@@ -27,7 +27,7 @@
 from bigdl.orca.learn.bigdl import Estimator
 from bigdl.dllib.optim.optimizer import Adam, SGD, ValidationSummary
 from bigdl.dllib.keras import layers as ZLayer
-from bigdl.dllib.keras.models import Model as ZModel
+from bigdl.dllib.keras.model import Model as ZModel
```
Comment (jason-dai) [inline python/orca/test/bigdl/orca/learn/spark/test_estimator_for_bigdl.py:30]: This is not correct; it should be either `from bigdl.dllib.keras import Model` or `from bigdl.dllib.keras.models import Model`

Comment (dding3) [reply python/orca/test/bigdl/orca/learn/spark/test_estimator_for_bigdl.py:30]: This pr is fix the exceptions in nightly build jekins. The pr for package rename to `from bigdl.dllib.keras import Model` is on progress.

**Pull request merged**